aea/package_manager/v1.py

```diff
@@ -272,7 +273,9 @@ def sync(
         return self
 
     def update_package_hashes(
-        self, selector_prompt: Callable[[], str]
+        self,
+        selector_prompt: Optional[Callable[[], str]] = None,
+        skip_missing: bool = False,
```

Comment:
@solarw Selector prompt is optional, if you don't provide a selector prompt callable it'll raise error if you don't set `skip_missing` to `True` explicitly.

Reply:
On the cli, you can provide `--skip-missing` flag to skip the packages which are missing from the `packages.json`

Reply:
@angrybayblade nice, thanks!